Lib/modulefinder.py

```diff
@@ -160,14 +160,14 @@ def msgout(self, *args):
 
     def run_script(self, pathname):
         self.msg(2, "run_script", pathname)
-        with open(pathname) as fp:
+        with open(pathname, "rb") as fp:
```

Comment:
These should be changed to use `io.open_code` anyway (which always opens with `rb`). Could you make that change?